src/contracts/libraries/ChainConfig.sol

```diff
@@ -0,0 +1,251 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.28;
+
+/// @title ChainConfig
+/// @notice Central configuration library for chain-specific deployment parameters
+/// @dev Single source of truth for all chain-specific configurations
+library ChainConfig {
+    struct ChainParameters {
+        uint256 escrowDuration;
+        address l2GasCalculator;
+        uint256 atlasSurchargeRate;
+        uint256 bundlerSurchargeRate;
+        string name;
+    }
+
+    /// @notice Get chain-specific parameters based on chain ID
+    /// @param chainId The blockchain chain ID
+    /// @return params The chain-specific parameters
+    function getChainParameters(uint256 chainId) internal pure returns (ChainParameters memory params) {
+        // Mainnet chains
+        if (chainId == 1) {
+            // Ethereum Mainnet
+            return ChainParameters({
+                escrowDuration: 10, // ~12 seconds * 10 blocks = 120 seconds
+                l2GasCalculator: address(0),
+                atlasSurchargeRate: 1000, // 10%
+                bundlerSurchargeRate: 1000, // 10%
+                name: "MAINNET"
+            });
+        } else if (chainId == 137) {
+            // Polygon
+            return ChainParameters({
+                escrowDuration: 64, // ~2 seconds * 32 blocks
+                l2GasCalculator: address(0),
+                atlasSurchargeRate: 500, // 5%
+                bundlerSurchargeRate: 2000, // 20%
+                name: "POLYGON"
+            });
+        } else if (chainId == 56) {
+            // BSC
+            return ChainParameters({
+                escrowDuration: 32, // ~3 seconds * 10 blocks
+                l2GasCalculator: address(0),
+                atlasSurchargeRate: 1000, // 10%
+                bundlerSurchargeRate: 1000, // 10%
+                name: "BSC"
+            });
+        } else if (chainId == 10) {
+            // Optimism
+            return ChainParameters({
+                escrowDuration: 30, // ~1 second * 30 blocks = 30 seconds
+                l2GasCalculator: address(0), // Will be deployed and set
+                atlasSurchargeRate: 1000, // 10%
+                bundlerSurchargeRate: 1000, // 10%
+                name: "OP MAINNET"
+            });
+        } else if (chainId == 42_161) {
+            // Arbitrum One
+            return ChainParameters({
+                escrowDuration: 128, // ~250ms * 128 blocks = 32 seconds
+                l2GasCalculator: address(0), // Will be deployed and set
+                atlasSurchargeRate: 1000, // 10%
+                bundlerSurchargeRate: 1000, // 10%
+                name: "ARBITRUM"
+            });
+        } else if (chainId == 8453) {
+            // Base
+            return ChainParameters({
+                escrowDuration: 30, // ~1 second * 30 blocks = 30 seconds
```

Comment:
This needs to be larger. See how fast blocks are added here: https://app.hyperliquid.xyz/explorer definitely not 1s